bloges56/DateSim
# Fd rhythm updates

## Description
- The rhythm game is completely different. 

## How to Test
1. Go to SceneManager Scene and hit play
3. Hit start game
4. Name scene should load. 
5.In the arcade, play the rhythm game



## Checklist
- [x] Does the rhythm game now contain sprite?
- [x] Does the # of lives go down if you miss one? 
- [x] Do you get a point for selecting the correct key
- [x] Do the restart and exit buttons work when failing to beat the level? 
- [x] Is everything else other than the rhythm game still working as intended?
